src/melee.cpp

```diff
@@ -769,12 +772,12 @@ double Character::crit_chance( float roll_hit, float target_dodge, const item &w
         // chance_triple and chance_double are mutually exclusive probabilities and can just
         // be added together.
         melee::melee_stats.double_crit_count += 1;
-        melee::melee_stats.double_crit_chance += chance_double + chance_triple;
-        return chance_triple + chance_double;
+        melee::melee_stats.double_crit_chance += chance_double + chance_triple + ma_buff_crit_chance;
```

Comment:
why not just simplify everyone's life and add ma_buff_crit_chance to chance_triple when chance_triple is first defined?

Reply:
chance_triple is used in the calculation of chance_double further down by subtracting 3 * chance_triple. I don't understand why the formula is set up like this but adding the ma_buff ahead of time would reduce the value of chance_double, making the buff less effective.

Reply:
Oh, right.  Good enough.